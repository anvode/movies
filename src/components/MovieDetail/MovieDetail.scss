.movie-detail {
    &__title {
        background-color: var(--color-primary);
        margin: 0;
        padding: 1rem 0;
        text-align: center;    }

    &__content {
        display: grid;
        grid-gap: 30px;
        grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
    }

    &__image {
        height: auto;
        max-width: 100%;
    }
}
