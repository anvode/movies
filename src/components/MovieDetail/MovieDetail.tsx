import * as React from 'react';

import './MovieDetail.scss';

export interface MovieDetailProps {}

const MovieDetail: React.FC<MovieDetailProps> = () => {

    return <>
       MovieDetail
    </>;
};

export default MovieDetail;