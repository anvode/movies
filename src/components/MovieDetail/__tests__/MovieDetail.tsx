import React from 'react';
import Store from '../../../redux/store';
import { render } from '@testing-library/react';
import MovieDetail from '../MovieDetail';
import {Provider} from 'react-redux';
import { BrowserRouter } from 'react-router-dom';

describe('MovieDetail', () => {
    it('renders without crashing', () => {
        const { asFragment } = render( <Provider store={Store}><BrowserRouter><MovieDetail /></BrowserRouter></Provider>);
        expect(asFragment()).toMatchSnapshot();
    });
});