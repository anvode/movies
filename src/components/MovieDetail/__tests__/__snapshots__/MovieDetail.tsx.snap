// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetail renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="main"
  >
    <div
      class="container"
    >
      <h1
        class="movie-detail__title"
      >
        No Movie Selected
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
